src/edu/saddleback/tictactoe/decision/DepthEvaluator.java

```diff
@@ -0,0 +1,122 @@
+package edu.saddleback.tictactoe.decision;
+
+import edu.saddleback.tictactoe.board.Board;
+import edu.saddleback.tictactoe.board.GamePiece;
+
+public class DepthEvaluator implements StaticEvaluator {
+
+
+    private int Winner(Board b){
```

Comment:
camelCase please

Reply:
also, can you javadoc these please? not 100% sure what each of these do by the name alone.